docs/troubleshooting/_common/enable-consistent-topology.rst

```diff
@@ -0,0 +1 @@
+Perform :doc:`the procedure for enabling consistent topology changes </upgrade/upgrade-opensource/upgrade-guide-from-5.4-to-6.0/enable-consistent-topology>`.
```

Comment:
I dislike that we need a separate file just for this.
We should implement a directive like `scylladb_include_flag` but one that allows writing **inline** what the different variants of the text are (for OSS and for Enterprise)